committer/src/config.rs

```diff
@@ -35,8 +35,35 @@ pub struct EthConfig {
     /// Ethereum address of the fuel chain state contract.
     pub state_contract_address: Address,
     /// The number of fuel blocks between ethereum commits. If set to 1, then every block should be pushed to Ethereum.
+    #[serde(
+        default = "default_commit_interval",
+        deserialize_with = "deserialize_commit_interval"
```

Comment:
Why use a custom deserializing fn when serde can handle it out of the box? Something weird around providing defaults it seems. Both with `default = ` and as a fallback for the deserialization.

In any case we should probably remove the whole parameter.